spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -53,57 +53,58 @@
 import org.springframework.messaging.support.GenericMessage;
 
 /** {@link PubSubInboundChannelAdapter} unit tests. */
-@RunWith(MockitoJUnitRunner.class)
-public class PubSubInboundChannelAdapterTests {
+@ExtendWith(MockitoExtension.class)
+@ExtendWith(OutputCaptureExtension.class)
+class PubSubInboundChannelAdapterTests {
 
   private TestUtils.TestApplicationContext context = TestUtils.createTestApplicationContext();
 
   PubSubInboundChannelAdapter adapter;
 
   static final String EXCEPTION_MESSAGE = "Simulated downstream message processing failure";
 
-  /** Output capture for validating warning messages. */
-  @Rule public OutputCaptureRule output = new OutputCaptureRule();
-
   @Mock PubSubSubscriberOperations mockPubSubSubscriberOperations;
 
   @Mock MessageChannel mockMessageChannel;
 
   @Mock ConvertedBasicAcknowledgeablePubsubMessage mockAcknowledgeableMessage;
 
-  @Before
+  @BeforeEach
   @SuppressWarnings("unchecked")
-  public void setUp() {
+  void primarySetup() {
 
     this.adapter =
         new PubSubInboundChannelAdapter(this.mockPubSubSubscriberOperations, "testSubscription");
     this.adapter.setOutputChannel(this.mockMessageChannel);
     this.adapter.setBeanFactory(this.context);
 
+  }
+
+  void secondarySetup() {
```

Comment:
Could you rename this to reflect the purpose -- something like `setupSubscribeAndConvert()`?